core/vm/evm.go

```diff
@@ -335,6 +338,15 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 		err = ErrContractAddressCollision
 		return nil, common.Address{}, 0, err
 	}
+	// Check whether the init code size has been exceeded.
+	if evm.config.HasEip3860(evm.chainRules) && len(codeAndHash.code) > params.MaxInitCodeSize {
```

Comment:
Wonder if brought in, more EIPs with this merge. Have to check as EIP3860 is not one of the EOF ones

Reply:
LGTM.

The checklist doc cites EIP-3860 as a prerequisite for EOF, and the EIP itself cites EIP-3670 (EOF - Code Validation) at a motivation:

> 2. To have a cost system which is extendable in the future (i.e. for proposals like [EIP-3670](https://eips.ethereum.org/EIPS/eip-3670)).

https://notes.ethereum.org/@ipsilon/eof1-checklist# 
https://eips.ethereum.org/EIPS/eip-3860